src/azul/project/hca/transformers.py

```diff
@@ -545,12 +545,19 @@ def _get_accumulator(self, field) -> Optional[Accumulator]:
 
 class CellSuspensionAggregator(GroupingAggregator):
 
+    def _transform_entity(self, entity: JSON) -> JSON:
+        return dict(document_id=entity['document_id'],
```

Comment:
```diff
Index: src/azul/project/hca/transformers.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/azul/project/hca/transformers.py	(date 1560328174000)
+++ src/azul/project/hca/transformers.py	(date 1560327966547)
@@ -546,11 +546,8 @@
 class CellSuspensionAggregator(GroupingAggregator):
 
     def _transform_entity(self, entity: JSON) -> JSON:
-        return dict(document_id=entity['document_id'],
-                    total_estimated_cells=(entity['document_id'], entity['total_estimated_cells']),
-                    selected_cell_type=entity['selected_cell_type'],
-                    organ=entity['organ'],
-                    organ_part=entity['organ_part'])
+        return dict(**entity,
+                    total_estimated_cells=(entity['document_id'], entity['total_estimated_cells']))
 
     def _group_keys(self, entity) -> Iterable[Any]:
         return entity['organ']
```

Reply:
`dict()` doesn't allow duplicate dictionary keys, but `{}` does so I'll use that

Reply:
It does

```
dict(dict(a=1), a=2)
{'a': 2}
```

but the ** shouldn't have been in my diff.

However, what you did is fine, too.